_overviews/scala3-macros/tutorial/reflection.md

```diff
@@ -36,16 +38,35 @@ def f(x: Expr[Int])(using Quotes): Expr[Int] =
 
 This will import all the types and modules (with extension methods) of the API.
 
-The full imported API can be found here: [Reflection](https://dotty.epfl.ch/api/scala/quoted/Quotes$reflectModule.html?query=trait%20reflectModule)
+## How to navigate the API
+
+The full imported API can be found in the [API documentation for `scala.quoted.Quotes.reflectModule`][reflection doc].
+Unfortunately, at this stage, this automatically generated documentation is not very easy to navigate.
+
+The most important element on the page is the hierarchy tree which provides a synthetic overview of the subtyping relationships of
+the types in the API. For each type `Foo` in the tree:
 
-For example to find what is a `Term`, we can see in the hierarchy that it is a subtype of `Statement` which is a subtype of `Tree`.
-If we look into `TermMethods` we will find all the extension methods that are defined for `Term` such as `Term.tpe` which returns a `Type`.
-As it is a subtype of `Tree` we can also look into the `TreeMethods` to find more methods such as `Tree.pos`.
-Each type is also a module with some _static-ish_ methods, for example in the [`TypeReprModule`](https://dotty.epfl.ch/api/scala/quoted/Quotes$reflectModule$TypeReprModule.html) we can find the method `TypeRepr.of[T]` which will create an instance of `Type` containing `T`.
+ - the object `FooMethods` contains the methods available for `Foo`
+ - the object `FooModule` contains some _static-ish_ methods, most notably constructors (`apply/copy`) and the `unapply` method which provides the extractor(s) required for pattern matching.
+ - For all types `Upper` such that `Foo <: Upper`, the methods defined in `UpperMethods` are available on `Foo` as well.
 
+For example [`TypeBounds`](https://dotty.epfl.ch/api/scala/quoted/Quotes$reflectModule.html#TypeBounds-0), a subtype of `TypeRepr`, represents a type tree of the form `T >: L <: U`: a type `T` which is a super type of `L`
```

Comment:
```suggestion
For example [`TypeBounds`](https://dotty.epfl.ch/api/scala/quoted/Quotes$reflectModule.html#TypeBounds-0), a subtype of `TypeRepr`, represents a type of the form `T >: L <: U`: a type `T` which is a super type of `L`
```

Reply:
I think it might be confusing to call `TypeRepr` when we have `TypeTree` around